Toolkit/ArcGISToolkit/AR/ArcGISARView.swift

```diff
@@ -0,0 +1,527 @@
+        return config
+        }() {
+        didSet {
+            // Start tracking using the new configuration.
+            startTracking()
+        }
+    }
+
+    /// The viewpoint camera used to set the initial view of the sceneView instead of the devices GPS location via the locationManager.
+    public var originCamera: AGSCamera?
+    
+    /// The translation factor used to support a table top AR experience.
+    public var translationTransformationFactor: Double = 1.0
+    
+    // We implement ARSessionDelegate methods, but will use `sessionDelegate` to forward them to clients.
+    weak open var sessionDelegate: ARSessionDelegate?
+    
+    // We implement SCNSceneRendererDelegate methods, but will use `scnSceneRendererDelegate` to forward them to clients.
+    weak open var scnSceneRendererDelegate: SCNSceneRendererDelegate?
+
+    // MARK: private properties
+    
+    /// Whether to display the camera image or not.
+    private var renderVideoFeed = true
+    
+    /// Used to determine the device location when originCamera is not set.
+    private lazy var locationManager: CLLocationManager = {
+        let lm = CLLocationManager()
+        lm.desiredAccuracy = kCLLocationAccuracyBest
+        lm.delegate = self
+        return lm
+    }()
+    
+    /// Initial location from locationManager.
+    private var initialLocation: CLLocation?
+    
+    /// Current horizontal accuracy of the device.
+    private var horizontalAccuracy: CLLocationAccuracy = .greatestFiniteMagnitude;
+    
+    /// The intial camera position and orientation whether it was set via originCamera or the locationManager.
+    private var initialTransformationMatrix = AGSTransformationMatrix()
+    
+    /// Whether `ARKit` is supported on this device.
+    private var isSupported = false
+    
+    /// Whether the client has been notfiied of start/failure.
+    private var notifiedStartOrFailure = false
+    
+    /// Used when calculating framerate.
+    private var lastUpdateTime: TimeInterval = 0
+    
+    // A quaternion used to compensate for the pitch beeing 90 degrees on `ARKit`; used to calculate the current device transformation for each frame.
+    let compensationQuat:simd_quatd = simd_quaternion((sin(45 / (180 / .pi))), 0, 0, (cos(45 / (180 / .pi))))
+    
+    /// The quaternion used to represent the device orientation; used to calculate the current device transformation for each frame; defaults to landcape-left.
+    var orientationQuat:simd_quatd = simd_quaternion(0, 0, 1, 0)
+    
+    // MARK: Initializers
+    
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        sharedInitialization()
+    }
+    
+    required public init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        sharedInitialization()
+    }
+    
+    /// Initializer used to denote whether to display the live camera image.
+    ///
+    /// - Parameter renderVideoFeed: whether to dispaly the live camera image.
```

Comment:
I would suggest starting parameter descriptions with an uppercase letter as well. Also, there's a typo:

```suggestion
    /// - Parameter renderVideoFeed: Whether to display the live camera image.
```